src/themes/shared/animation.js

```diff
@@ -8,13 +8,22 @@ LICENSE file in the root directory of this source tree.
 import type {AnimationT} from '../types.js';
 
 const animation: AnimationT = {
-  timing100: '0.25s',
-  timing400: '0.4s',
-  timing700: '0.6s',
-  timing1000: '1s',
-  easeOutCurve: 'cubic-bezier(.2, .8, .4, 1)',
+  timing100: '100ms',
+  timing200: '200ms',
+  timing300: '300ms',
+  timing400: '400ms',
+  timing500: '500ms',
+  timing600: '600ms',
+  timing700: '700ms',
+  timing800: '800ms',
+  timing900: '900ms',
+  timing1000: '1000ms',
   easeInCurve: 'cubic-bezier(.8, .2, .6, 1)',
```

Comment:
@sandgraham @andersonba - should we also update these default ease values to match quintic since that is our default in the guidelines? 

Making this update could require adjusting some of the timing where they are being used. It could be something done in a separate PR to keep them more atomic.

Reply:
I'm for updating them - no preference on if a new pr is warranted.

Reply:
> @sandgraham @andersonoliveira - should we also update these default ease values to match quintic since that is our default in the guidelines?
> 
> Making this update could require adjusting some of the timing where they are being used. It could be something done in a separate PR to keep them more atomic.

@chrisrauh I'll update the timing and documentation first. After that will look into the easing and get a new PR. Thanks

Reply:
Branch updated.